libs/island-ui/core/src/lib/Button/Button.tsx

```diff
@@ -57,11 +57,13 @@ export const Button = forwardRef<
             [styles.size[size]]:
               variant !== 'utility' &&
               !circle &&
-              !(variant === 'text' && size === 'small'),
+              !(variant === 'text' && size === 'small') &&
```

Comment:
Is this condition needed now?

Reply:
No, you're right. Good catch. It's now been removed 🙏